playbooks/init/facts.yml

```diff
@@ -107,7 +103,7 @@
       - ostree
       - runc
       when:
-      - l_any_system_container | bool
+      - openshift_use_system_containers or (openshift_use_etcd_system_container | default(False)) or (openshift_use_openvswitch_system_container | default(False)) or (openshift_use_node_system_container | default(False)) or (openshift_use_master_system_container | default(False))
```

Comment:
Looks like this still needs `default` and probably also `bool` for openshift_use_system_containers.

`(openshift_use_system_containers | default(false)) | bool`